zbus/src/connection/socket/unix.rs

```diff
@@ -353,10 +360,12 @@ fn get_unix_peer_creds_blocking(fd: RawFd) -> io::Result<crate::fdo::ConnectionC
         target_os = "netbsd"
     ))]
     {
-        let uid = nix::unistd::getpeereid(fd).map(|(uid, _)| uid.into())?;
-        // FIXME: Handle pid fetching too.
```

Comment:
did this comment get accidentally deleted? At least it's not being addressed in this commit.

Reply:
It did. Added it again.